### DISCUSSION TITLE
How to Improve my Site SEO Rank

### DISCUSSION THREAD

**Discussion by RudraSen2** [facebook/docusaurus#8127] [General]
Hello,
I have a docusaurus site (Link below). I want to improve its Performance and Overall SEO (If Possible and Performance Rank given below).
**Link** - https://complabs.in
**PageSpeed Insights** - https://pagespeed.web.dev/report?url=https://complabs.in
**SEO Report** - [Comp Labs SEO Report.pdf](https://github.com/facebook/docusaurus/files/9639106/All.issues.Complabs.pdf)
**Source** - https://github.com/Comp-Labs/comp-labs-website

Please tell me how to improve it as I cannot do so. Any help would be appreciated 😄!

**Comment by johnnyreilly** [comment]
I suggest running Lighthouse against your site and following the SEO points that it raises.  Performance advice I'm less clear on, in my own experience Docusaurus doesn't have the best performance out of the box.  Would love it to be better.

**Reply by RudraSen2** [reply]
I ran the Lighthouse report before and I got the problems also but I don't know how to fix it.

**Comment by slorber** [comment]
We are currently not the best in term of hydration performance, js/css size etc but it should probably be good enough to not be significantly penalized by crawlers.
We'll improve performance later with React 18, server components, partial hydration etc

Note that Docusaurus do better than you on the lighthouse score, so maybe you did something wrong on your landing page (probably too many complex inlined svgs blocking the first render):
https://pagespeed.web.dev/report?url=https%3A%2F%2Fdocusaurus.io%2F&form_factor=mobile

I think we do well on the technical SEO side (metadata etc).
Now for content-based SEO it's your responsibility.

I'm not sure what to advise you here, the low-hanging fruits are more likely to produce better content and get backlinks, and is outside the scope of Docusaurus
